examples/callback/example_callback.cpp

```diff
@@ -53,12 +53,6 @@ struct AccessTraits<NearestToOrigin, PredicatesTag>
 };
 } // namespace ArborX
 
-struct PairIndexDistance
-{
-  int index;
-  float distance;
-};
-
```

Comment:
Forgot to remove in #366 ?

Reply:
Yes, discovered during the merge. Thought it was innocent enough change to not warrant a separate commit or PR.